### PR TITLE
[oadp-1.3] bump kubevirt to 0.6.2

### PR DESCRIPTION
**Pull request by kaovilai**
Signed-off-by: Tiger Kaovilai <tkaovila@redhat.com>

Why the changes were made
We're using a VERY old kubevirt plugin in prow tests and dev setup

How to test the changes made
make deploy-olm and check the deployment

### CODE REVIEW

File: config/manager/manager.yaml [hunk 1]
```diff
@@ -51,7 +51,7 @@ spec:
           - name: RELATED_IMAGE_VELERO_PLUGIN_FOR_CSI
             value: quay.io/konveyor/velero-plugin-for-csi:oadp-1.3
           - name: RELATED_IMAGE_KUBEVIRT_VELERO_PLUGIN
-            value: quay.io/konveyor/kubevirt-velero-plugin:v0.2.0
+            value: quay.io/konveyor/akubevirt-velero-plugin:v0.6.2
```
Comment (weshayutin) [inline config/manager/manager.yaml:54]: typo

File: pkg/common/common.go [hunk 2]
```diff
@@ -66,7 +66,7 @@ const (
 	GCPPluginImage       = "quay.io/konveyor/velero-plugin-for-gcp:latest"
 	CSIPluginImage       = "quay.io/konveyor/velero-plugin-for-csi:latest"
 	RegistryImage        = "quay.io/konveyor/registry:latest"
-	KubeVirtPluginImage  = "quay.io/konveyor/kubevirt-velero-plugin:v0.2.0"
+	KubeVirtPluginImage  = "quay.io/konveyor/akubevirt-velero-plugin:v0.6.2"
```
Comment (weshayutin) [inline pkg/common/common.go:69]: typo

File: bundle/manifests/oadp-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -858,7 +858,7 @@ spec:
                 - name: RELATED_IMAGE_VELERO_PLUGIN_FOR_CSI
                   value: quay.io/konveyor/velero-plugin-for-csi:oadp-1.3
                 - name: RELATED_IMAGE_KUBEVIRT_VELERO_PLUGIN
-                  value: quay.io/konveyor/kubevirt-velero-plugin:v0.2.0
+                  value: quay.io/konveyor/akubevirt-velero-plugin:v0.6.2
```
Comment (mateusoliveira43) [inline bundle/manifests/oadp-operator.clusterserviceversion.yaml:861]: typo?

**Review by weshayutin — APPROVED** [review]
/LGTM

**Review by shubham-pampattiwar — APPROVED** [review]

**Review by rayfordj — APPROVED** [review]
/lgtm

**Comment by mateusoliveira43** [comment]
I saw some errors I never saw: api server may have been down. Should we investigate that?

**Comment by kaovilai** [comment]
Error came from https://github.com/openshift/velero/pull/315

**Comment by weshayutin** [comment]
> Error came from [openshift/velero#315](https://github.com/openshift/velero/pull/315)

uh oh..  Tiger please investigate this.

**Comment by kaovilai** [comment]
Takes about 20 sec after creation to fail.
```
2024/06/20 21:04:31 Creating backup mysql-csi-e2e-90f0f6b7-2f48-11ef-a75a-0a580a812821 for case mysql-csi-e2e
2024/06/20 21:04:41 backup phase: InProgress
2024/06/20 21:04:52 backup phase: Failed
```

**Comment by kaovilai** [comment]
fix fyi: https://github.com/openshift/velero/pull/324/

**Review by mateusoliveira43 — CHANGES_REQUESTED** [review]
Also had the same problem here: If I add kubevirt plugin in DPA, velero container never succeeds with `Init:CrashLoopBackOff` with kubevirt plugin restarting 6 times in 10 minutes

**Pull request merged**